ManualSetup/6_InsertData_WEBPAGE_ADDRESS.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO [WEBPAGE_ADDRESS] (nIndex, strWebPageAddress) VALUES
+(N'1         ', N'sohu.com  ')
```

Comment:
With the previous column changes, WEBPAGE_ADDRESS.strWebPageAddress is the only one that needs trimming.